docs/actions.rst

```diff
@@ -51,6 +44,9 @@ All actions in Crud, even your own, should inherit from the
 This class is ``abstract`` and provides numerous auxiliary methods which can be
 useful for you both as a developer as an action creator.
 
+Where you choose to put your custom Crud actions is up to you, but using ``src/Crud`` is a good place. Remember to
```

Comment:
**Solved:** silly me... I forgot to include `Action` suffix so trying to the `new()` call on L699 of CrudComponent failed. 

**Feedback 1:**
Using single slashes totally works for me as well:

```php
->mapAction('changePassword', '\App\Crud\Action\ChangePassword');
```
**Feedback 2:**
Looking at the code I believe the advised directories should mimic Crud and be:

- `Action` (without the trailing s)
- `Listener` (without the trailing s)